pkg/test/util/shell.go

```diff
@@ -0,0 +1,68 @@
+// Copyright 2018 Istio Authors
```

Comment:
I thought we already had shell utilities?

Reply:
oof good catch ... didn't notice that. Fixed to use the existing shell functions.